src/app/util/clipboard.cpp

```diff
@@ -805,6 +824,13 @@ ImageRef Clipboard::getImage(Palette* palette)
   return m_data->image;
 }
 
+bool Clipboard::isImageAvailable() const
+{
+  return m_data->image &&
+    m_data->image->width() > 0 &&
+    m_data->image->height() > 0;
+}
+
```

Comment:
Probably this should go to some CliClipboardManager (which could implement an InputChain and be the default one) to handle all the cut/paste/copy logic for scripts (when UI and editors are not available). Not sure but I think it might be a possible design.